### ISSUE TITLE
How do you exactly install this?

### ISSUE THREAD

**Issue by Ckt33** [zeyanthecoder/Teron-AI#2]
I dont get how your supposde to run it on my computer can someone give me an step by step guide?

**Comment by ghost** [comment]
For installing this program you need to have python installed in your system

**Comment by Ckt33** [comment]
Yes already have that but i dont get the instructions on how to actually run it

**Comment by ghost** [comment]
All the instructions are given in the git description

**Comment by Ckt33** [comment]
in what description?

**Comment by Ckt33** [comment]
i dont get what i must install theres just install rpm instruction idk what i should install honestly?

**Comment by ghost** [comment]
You just have to install the modules using pip

**Comment by Ckt33** [comment]
which modules can u be more concrete?

**Comment by ghost** [comment]
https://youtu.be/3wo6RMzr6FU?si=zkiyxLl8qNK1ZVSd
This is the tutorial video
